README.md

```diff
@@ -21,34 +21,34 @@ Download
 
 [<img src="https://f-droid.org/badge/get-it-on.png" height="75">][fdroid]
 
-You can download the latest version from GitHub
+The latest version can be downloaded from GitHub
 - [Download latest release](https://niccokunzmann.github.io/download_latest/mundraub-release-unsigned.apk)
 - [Download latest debug version](https://niccokunzmann.github.io/download_latest/mundraub-release-unsigned.apk)
 
 Contribute
 ----------
 
 Contributions are welcome.
-
-- If you like to have your language included, you can translate the app yourself on [Transifex][tx].
```

Comment:
I like active voice. That is a in harmony with "positive action language" and marks the person putting the energy into the action - the subject.

> Practicing non-violent communication means using positive action language rather than negative action language.
- https://pairedlife.com/relationships/Compassionate-Communication-The-key-to-reducing-conflict-and-improving-your-relationships

It is a way of shaping my thinking towards peace. That are some thoughts, in German: http://niccokunzmann.github.io/blog/2018-05-23/newspeak-vermeiden

Also, I see, I do not use it consistently.